chapter2/tests/t19/terraform/base/main.tf

```diff
@@ -0,0 +1,23 @@
+module "nginx" {
+  source                       = "../_modules/nginx"
+  container_image              = var.nginx.image
+  container_name               = var.nginx.container_name != "" ? var.nginx.container_name : null
```

Comment:
make all validation in the inner module, not in the outer invocation. That way if your module REQUIRE the name - you will get error

Reply:
> make all validation in the inner module, not in the outer invocation. That way if your module REQUIRE the name - you will get error

@dmitry-mightydevops
Done